### DISCUSSION TITLE
😫 This drives me crazy, can anyone explain why this happened?

### DISCUSSION THREAD

**Discussion by ntnyq** [unocss/unocss#4279] [General]
`unocss-preset-magicss` in https://unocss.dev/presets/community is not colored in `var(----vp-c-brand-1)`.

But why? It's just a simple link as others above and bellow.

And if you delete any single char from the url `https://github.com/unpreset/unocss-preset-magicss`, It will work as expected.

Why? Why again? 😫😫😫

## Screenshot

<img width="785" alt="Screenshot 2024-11-22 at 16 11 52" src="https://github.com/user-attachments/assets/ed9e9a9e-400d-4851-ac98-ca6217b7c839">

## Environment

**Google Chrome Version 131.0.6778.71 (Official Build) (arm64)**

Firefox, Safari, MS Edge, even in  Chrome incognito mode, it works fine.

**Comment by ntnyq** [comment]
@zyyv 

Can you repro this? Any idea?

**Reply by zyyv** [reply]
I think this is determined by the GPU capabilities of the computer. A large number of our DOM elements use CSS variables to change colors, but the computer's rendering capabilities cannot keep up.

On my inter chip mac, it even performs worse.

But it performs well on my M chip mac.

**Comment by antfu** [comment]
Works for me on both Chrome, Safari and Firefox - sounds like might caused by your extension.

**Reply by ntnyq** [reply]
Yes, disabled all extension solve the problem. 
But after enabled all extension again, the issue can't be reproduced neither.
🤣
